tools/validators/instance_validator/tests/fake_instances/GOOD/building_translation_fields.yaml

```diff
@@ -15,12 +15,12 @@
 CHWS_WDT-17-GUID:
   type: HVAC/CHWS_WDT
   code: CHWS_WDT-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
done.